helm/generic-node/templates/generic-node.yaml

```diff
@@ -0,0 +1,199 @@
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Values.name}}
+  template:
+    metadata:
+      labels:
+        app: {{.Values.name}}
+        testnet: {{.Values.testnetName}}
+        role: seed
+        class: "generic-node"
+        version: {{.Values.image.tag | quote}}
+        syncStatus: INIT
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: {{.Values.ports.metrics | quote}}
+        prometheus.io/path: '/metrics'
+    spec:
+      {{- if .Values.libp2p.staticKeypair.enabled}}
+      initContainers:
+      - name: libp2p-init-my-service
+        image: busybox:1.28
+        command: ['sh', '-c', '/bin/cp /libp2p-keys/* /root/libp2p-keys && /bin/chmod -R 0700 /root/libp2p-keys/']
+        volumeMounts:
+        - name: source-libp2p-keys
+          mountPath: /libp2p-keys 
+        - name: destination-libp2p
+          mountPath: /root/libp2p-keys
+      {{- end }}
+      containers:
+      - name: daemon
+        image: "{{.Values.image.repository }}:{{.Values.image.tag}}"
+        imagePullPolicy: {{.Values.image.pullPolicy}}
+        {{- if .Values.features.customEntrypoint.enabled}}
+        command: [ {{ .Values.features.customEntrypoint.command}} ]
+        {{- end }}
+
+        resources:
+          limits:
+          requests:
+            memory: "32Gi"
+            cpu: "12"
+        args:
+        - "daemon"
+        - "--log-level"
+        - "Debug"
+        - "--log-json"
+        - "--log-snark-work-gossip"
+        - "true"
+        - "--config-directory"
+        - "/root/.features-config"
+        - "--client-port"
+        - "$(DAEMON_CLIENT_PORT)"
+        - "--rest-port"
+        - "$(DAEMON_REST_PORT)"
+        - "--external-port"
+        - "$(DAEMON_EXTERNAL_PORT)"
+        - "--metrics-port"
+        - "$(DAEMON_METRICS_PORT)"
+        - "--enable-peer-exchange"
+        - "true"
+        - "--peer-list-url"
+        - "https://storage.googleapis.com/seed-lists/{{.Values.testnetName}}_seeds.txt"
+        {{- if .Values.libp2p.staticKeypair.enabled}}
+        - "--discovery-keypair"
+        - "/root/libp2p-keys/key"
+        {{- end }}
+        {{- if .Values.libp2p.seed.enabled}}
+        - "--seed"
+        {{- end }}
+        {{- if .Values.features.runtimeConfig.enabled}}
+        - "-config-file"
+        - "/config/daemon.json"
+        {{- end }}
+        {{- if .Values.features.logTxnPoolGossip.enabled}}
+        - "-log-txn-pool-gossip"
+        - "true"
+        {{- end }}
+        {{- if .Values.features.maxConnections}}
+        - "--max-connections"
+        - {{.Values.features.maxConnections | quote}}
+        {{- end }}
+        {{- range .Values.features.seedPeers}}
+        - "-peer"
+        - {{ . | quote }}
+        {{- end }}
+        {{-  if .Values.features.uploadBlocksToGCloud.enabled}}
+        - "-upload-blocks-to-gcloud"
+        - "true"
+        {{- end }}
+        env:
+        - name: RAYON_NUM_THREADS
+          value: "6"
```

Comment:
Make this configurable in .Values.resources.requests like .Values.resources.requests.rayonThreads